pkgs/development/python-modules/biothings-client/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  requests,
+}:
+let
+  finalAttrs = {
+    pname = "biothings-client";
+    version = "0.3.1";
+    pyproject = true;
+
+    src = fetchPypi {
+      inherit (finalAttrs) version;
+      pname = "biothings_client";
+      hash = "sha256-yXK/LgK2+cx49/L7xe8CzFb+T4oq3LiAHskC9KtwEeY=";
+    };
+
+    build-system = [ setuptools ];
+    dependencies = [ requests ];
+    pythonImportsCheck = [ "biothings_client" ];
+
+    meta = {
+      changelog = "https://raw.githubusercontent.com/biothings/biothings_client.py/refs/heads/master/CHANGES.txt";
+      description = "biothings_client is an easy-to-use Python wrapper to access any Biothings.api -based backend service.";
+      homepage = "https://github.com/biothings/biothings_client.py";
+      license = lib.licenses.bsd3;
+      maintainers = with lib.maintainers; [ rayhem ];
+      mainProgram = "biothings_client";
```

Comment:
Does it really have a CLI?

Reply:
It does not 😄 I fumbled through this by looking at another package; I think I've fixed this per the top-level `README.md`